### DISCUSSION TITLE
Actions failed to push to Github packages

### DISCUSSION THREAD

**Discussion by lei-splashtop** [community/community#32265] [Actions]
GitHub Actions build that uses a [http://ghcr.io](https://t.co/XjVMxgjc66) container has started failing with `denied: permission_denied: read_package`. We have permissive access for our GITHUB_TOKEN and haven't changed anything recently.
We discovered this problem at 7:30 yesterday and submitted a ticket. The result was that it took 5 hours to respond, and 8 hours to fix it.
Starting at 7:10 today, we have the same problem again!

Open here for a look, https://www.githubstatus.com/history
Guys, 11 Incidents in August 2022, 6 Incidents since September.

**Comment by sayhiben** [comment]
Also experiencing this issue. Seems similar to last night's incident(s).

**Reply by jundayou** [reply]
yes, but you can use PAT Token replace pipeline GITHUB_TOKEN, that's should work

**Reply by sayhiben** [reply]
That's not a solution when I have 100+ workflows impacted.
